daal4py/sklearn/neighbors/knn.py

```diff
@@ -30,6 +30,22 @@
 import logging
 
 
+def configure_algorithm(classifier, fptype, params):
```

Comment:
I see no reason return two algorithms here. I would write separately the functions:
```python
def training_algorithm(classifier, fptype, params):
...
```
and
```python
def prediction_algorithm(classifier, fptype, params):
...
```

Reply:
Fixed